source/extensions/quic_listeners/quiche/platform/quic_logging_impl.h

```diff
@@ -59,9 +59,18 @@
 #define CHECK(condition)                                                                           \
   QUIC_LOG_IF_IMPL(FATAL, ABSL_PREDICT_FALSE(!(condition))) << "CHECK failed: " #condition "."
 
+#define CHECK_GT(a, b) CHECK((a) > (b))
+#define CHECK_GE(a, b) CHECK((a) >= (b))
+#define CHECK_LT(a, b) CHECK((a) < (b))
+#define CHECK_LE(a, b) CHECK((a) <= (b))
+#define CHECK_NE(a, b) CHECK((a) != (b))
+#define CHECK_EQ(a, b) CHECK((a) == (b))
+
 #ifdef NDEBUG
 // Release build
-#define DCHECK(condition) QUIC_COMPILED_OUT_LOG()
+#define DCHECK(condition)                                                                          \
```

Comment:
I'm curious, what is the problem without this change?

Reply:
Got it. There are other macros defined to "QUIC_COMPILED_OUT_LOG" as well, maybe change "QUIC_COMPILED_OUT_LOG" to take a "condition" instead?

#define QUIC_COMPILED_OUT_LOG(condition)  QUIC_LOG_IMPL_INTERNAL(false && (condition), quic::NullLogStream().stream())

Reply:
done